lib/Property/ICalendar/DateTime.php

```diff
@@ -343,6 +343,7 @@ public function validate($options = 0)
         $messages = parent::validate($options);
         $valueType = $this->getValueType();
         $values = $this->getParts();
+        $value = null;
```

Comment:
it is highly unusual to initialize the variable which is used within a foreach. From my POV it would be better to move the try/catch into the foreach and break out of the loop in case of exception

Reply:
Ok, will fix this.